sklearn/kernel_approximation.py

```diff
@@ -619,10 +619,7 @@ def _get_kernel_params(self):
             if (self.gamma is not None or
                     self.coef0 is not None or
                     self.degree is not None):
-                warnings.warn(
-                    "Passing gamma, coef0 or degree to Nystroem when using a"
-                    " callable kernel is deprecated in version 0.19 and will"
-                    " raise an error in 0.21, as they are ignored. Use "
-                    "kernel_params instead.", DeprecationWarning)
+                raise ValueError("Don't pass gamma, coef0 or degree to "
+                                 "Nystroem if using a callable kernel.")
```

Comment:
The raise have to say `Use kernel_params instead`, or is not necessary?

Reply:
I think checking for this string is specific enough.